packages/cip2/src/RoundRobinRandomImprove/change.ts

```diff
@@ -3,14 +3,21 @@ import { orderBy } from 'lodash-es';
 import { ComputeMinimumCoinQuantity, TokenBundleSizeExceedsLimit } from '../types';
 import { InputSelectionError, InputSelectionFailure } from '../InputSelectionError';
 import { AssetQuantities, ValueQuantities, valueQuantitiesToValue } from '../util';
-import { assetQuantitySelector, getCoinQuantity, OutputWithTotals, UtxoSelection, UtxoWithTotals } from './util';
+import {
+  assetQuantitySelector,
+  assetTotalsQuantitySelector,
+  getCoinQuantity,
+  getCoinTotalsQuantity,
+  UtxoSelection,
+  UtxoWithTotals
+} from './util';
 
 type EstimateTxFeeWithOriginalOutputs = (utxo: CSL.TransactionUnspentOutput[], change: CSL.Value[]) => Promise<bigint>;
 
 interface ChangeComputationArgs {
   csl: CardanoSerializationLib;
   utxoSelection: UtxoSelection;
-  outputsWithTotals: OutputWithTotals[];
+  outputsQuantities: ValueQuantities[];
```

Comment:
```suggestion
  outputValues: Value[];
```

Reply:
We should stay close to the Ogmios type name, as we will soon be implementing it directly:
```
export interface Value {
    coins: Lovelace;
    assets?: {
        [k: string]: AssetQuantity;
    };
}
```

Reply:
I renamed the type we're using to `OgmiosValue`. When we implement this type directly I suggest we leave `OgmiosValue` as re-export for internal use, because having 2 types with same name is confusing (`CSL.Value` is the other one).